BoneReconstructionPlanner/BoneReconstructionPlanner.py

```diff
@@ -471,58 +477,96 @@ def setDefaultParameters(self, parameterNode):
     if not parameterNode.GetParameter("Invert"):

```

Comment:
move initialSpace and betweenSpace parameters here, to be parameters stored in the parameter node